### PR TITLE
78 saurabh add extra fields to create exercise page

### PR DESCRIPTION
**Pull request by saurabh21289**
electron-enabled-app/anchor/server/web/templates/exercise/create.handlebars relies on electron-enabled-app/anchor/server/web/public/scripts/exercise/create.js which depends on the schema which is a part of other PR. @shreyapandit please correct me if I am wrong. I'm running into errors when running the master branch and my own branch so am unable to check this

### CODE REVIEW

File: anchor/server/web/templates/userexercise/setting.handlebars [hunk 1]
```diff
@@ -21,6 +21,110 @@
...
+                   <option value="3">Head</option>
+                   <option value="20">Spine Shoulder</option>
+                   <option value="1">Spine Mid</option>
+                   <option value="0">Spine Base</option>
+                   <option value="4">Left Shoulder</option>
+                   <option value="5">Left Elbow</option>
+                   <option value="6">Left Wrist</option>
+                   <option value="7">Left Hand</option>
+                   <option value="21">Left Hand Tip</option>
+                   <option value="22">Left Thumb</option>
```
Comment (shreyapandit) [inline anchor/server/web/templates/userexercise/setting.handlebars]: @saurabh21289 These changes should not be here. As discussed we do not need to modify these values in setting. They should be in joiForm 
const schema = Joi.object({
  exerciseName: Joi.string().required(),
  description: Joi.string().required()
});
in the create script.

File: anchor/server/web/templates/userexercise/setting.handlebars [hunk 2]
```diff
@@ -21,6 +21,110 @@
...
+                   <option value="12">Left Hip</option>
+                   <option value="13">Left Knee</option>
+                   <option value="14">Left Ankle</option>
+                   <option value="15">Left Foot</option>
+                   <option value="8">Right Shoulder</option>
+                   <option value="9">Right Elbow</option>
+                   <option value="10">Right Wrist</option>
```
[the remark this answers is not in the document]
Comment (saurabh21289) [reply anchor/server/web/templates/userexercise/setting.handlebars]: Done.

File: anchor/server/web/templates/userexercise/setting.handlebars (hunk 1 above)
Comment (saurabh21289) [reply anchor/server/web/templates/userexercise/setting.handlebars]: Removed unwanted changes from this file

File: anchor/server/web/templates/exercise/index.handlebars [hunk 3]
```diff
@@ -14,6 +14,14 @@
                 <th>Exercise Name</th>
                 <th>Description</th>
                 <th>Created At</th>
+                <th>numSessions</th>
+                <th>numRepetition</th>
+                <th>impJoint</th>
+                <th>impAxis</th>
+                <th>direction</th>
+                <th>lowerJoint</th>
+                <th>upperJoint</th>
```
Comment (shreyapandit) [inline anchor/server/web/templates/exercise/index.handlebars]: @saurabh21289 what is type here for? Exercise as an entity does not have a type, a UserExercise on the other hand has a type - Reference or practice. We can remove this

File: anchor/server/web/partials/jointList.handlebars [hunk 4]
```diff
@@ -0,0 +1,29 @@
+<div class="form-group">
+    <select type="number" class="form-control" id="lowerJoint" aria-describedby="" placeholder="" value={{defaultLowerJoint}}>
+      <option value="2">Neck</option>
+      <option value="3">Head</option>
+      <option value="20">Spine Shoulder</option>
+      <option value="1">Spine Mid</option>
+      <option value="0">Spine Base</option>
+      <option value="4">Left Shoulder</option>
+      <option value="5">Left Elbow</option>
+      <option value="6">Left Wrist</option>
```
Comment (shreyapandit) [inline anchor/server/web/partials/jointList.handlebars]: @EvenGu you will have to make sure to parse these into integers before using them for your calculation.

File: anchor/server/web/templates/exercise/index.handlebars (hunk 3 above)
Comment (saurabh21289) [reply anchor/server/web/templates/exercise/index.handlebars]: "type" here was for Reference/Practice, I have removed it now as suggested

File: anchor/server/web/templates/exercise/create.handlebars [hunk 5]
```diff
@@ -9,6 +9,14 @@
...
               <div id="formFields"></div>
+              <label for="joint" class="label">Joint</label>
+              {{> jointList}}
+              <label for="lowerJoint" class="label">Axis</label>
+              {{> axis}}
+              <label for="lowerJoint" class="label">Ref Lower Joint</label>
+              {{> jointList}}
+              <label for="upperJoint" class="label">Ref Lower Joint</label>
+              {{> jointList}}
               <button id="create" type="submit" class="btn btn-primary">Create</button>
```
[the remark this answers is not in the document]
Comment (saurabh21289) [reply anchor/server/web/templates/exercise/create.handlebars]: Added as suggested

**Pull request merged**